### PR TITLE
Start building heroku/pack:20 and heroku/pack:20-build

### PR DESCRIPTION
**Pull request by edmorley**
The first step for Heroku-20 support is to start building and publishing the `heroku/pack:20` and `heroku/pack:20-build` stack images.

Once these are available, CNB authors will be able to test their buildpacks against the new stack, and in a later PR we'll start publishing the `heroku/buildpacks:20` builder with the updated CNBs.

For more on the remaining Heroku-20 tasks, see:
https://salesforce.quip.com/LbhlAFPd1rvR#fCAACA6FA2X

Closes [W-8270099](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008hLTWIA2/view).

### CODE REVIEW

File: Dockerfile.run
Comment (jkutner) [inline Dockerfile.run:13]: I don't want to block @edmorley on this, but switching to `heroku/heroku:20` might be a good opportunity for pack-images and our CNBs to move away from `stack = "heroku-20"` and towards `stack = "io.buildpacks.stacks.bionic"` plus mixins.

Comment (jkutner) [reply Dockerfile.run:13]: Example:
https://github.com/paketo-buildpacks/stacks/blob/main/bionic/cnb/run/Dockerfile#L13-L14

Comment (edmorley) [reply Dockerfile.run:13]: @jkutner Should I ask the CNB maintainers to hold off adding `stack = "heroku-20"` then? I've just filed a bunch of work items asking them to add it, and https://github.com/heroku/procfile-cnb/pull/5 has already merged.

Or is such a move something we can do in a non-breaking way for both heroku-18 and heroku-20 in the future? Since `builder.toml` contains versioned CNB URLs, it seems we can switch over the builder to use the new IDs in one go in the future, without breakage?

Comment (jkutner) [reply Dockerfile.run:13]: We should have the buildpacks add `heroku-20`, so those work items are 👌 We can migrate without breaking anything.

(Sidenote: Actually, we really want to move `io.buildpacks.stacks.focal`)

Comment (jabrown85) [reply Dockerfile.run:13]: If the referenced PRs aren't merged, we might consider adding `io.buildpacks.stacks.focal` and `io.buildpacks.stacks.bionic` as supported stacks in those buildpack PRs. It may save some time in the future and unless someone is building their own builder, it shouldn't have any consequences.

Comment (jkutner) [reply Dockerfile.run:13]: @jabrown85 but that would be a lie in most cases, because they also require some mixins. maybe better to wait until the buildpack owners can add both?

Comment (jabrown85) [reply Dockerfile.run:13]: ah, yeah. That is the part that was _super_ annoying when we tried to first make the switch to bionic. It's whack-a-mole trying to figure out which mixins you require to be present.

Comment (edmorley) [reply Dockerfile.run:13]: Yeah I think since we don't yet have the list of mixins, it would be best to hold off on buildpacks declaring support for `io.buildpacks.stacks.focal`.

Since the conversation about changing stack name/label has moved to email, and that we don't want to block on it, are we good to call this PR thread resolved, and to continue with this PR? :-)

Comment (jkutner) [reply Dockerfile.run:13]: Yes, we should move forward with this PR

**Review by jabrown85 — APPROVED** [review]

**Comment by edmorley** [comment]
I'm getting a:

> Merging is blocked
> The base branch requires all commits to be signed

I don't have GPG signing set up locally, but since I was going to use "squash and merge", GitHub automatically signs the commit that ends up on master, per:
https://docs.github.com/en/free-pro-team@latest/github/authenticating-to-github/about-commit-signature-verification#about-commit-signature-verification

As such the GitHub check seems broken?

**Comment by jabrown85** [comment]
> As such the GitHub check seems broken?

I noticed the same. We were told to turn that check on by security. I think you'll need to amend your commits with a gpg signature. Sorry :(

**Comment by edmorley** [comment]
No worries - good excuse to enable GPG signing locally too :-)

**Comment by edmorley** [comment]
Images published:
https://app.circleci.com/pipelines/github/heroku/pack-images/737/workflows/c627ad42-0928-4489-834b-f3d6e91d13ac
https://hub.docker.com/r/heroku/pack/tags/?page=1&name=20

**Pull request merged**
